app/src/main/java/infrmr/newsapp/github/com/ifrmr/ArticleArrayAdapter.java

```diff
@@ -0,0 +1,104 @@
+package infrmr.newsapp.github.com.ifrmr;
+
+import android.app.Activity;
+import android.content.Context;
+import android.text.Html;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.RelativeLayout;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+
+import de.keyboardsurfer.android.widget.crouton.Crouton;
+import de.keyboardsurfer.android.widget.crouton.Style;
+
+public class ArticleArrayAdapter extends BaseAdapter {
+
+    public static ArrayList<TheVergeXmlParser.Entry> articles = new ArrayList<TheVergeXmlParser.Entry>();
+    private static LayoutInflater inflater;
+    private final Context mContext;
+
+    public ArticleArrayAdapter(Context mContext) {
+        this.mContext = mContext;
+        inflater = LayoutInflater.from(mContext.getApplicationContext());
+
+    }
+
+    @Override
+    public int getCount() {
+        return articles.size();
+    }
+
+    @Override
+    public TheVergeXmlParser.Entry getItem(int i) {
+        return articles.get(i);
+    }
+
+    @Override
+    public long getItemId(int i) {
+        return i;
+    }
+
+    @Override
+    public View getView(int position, View convertView, ViewGroup parent) {
+
+        final ViewHolder holder;
+        final TheVergeXmlParser.Entry e = articles.get(position);
+
+        // Init view if null
+        if (null == convertView) {
```

Comment:
Why null == convertView and not convertView == null? It's standard convention to put the variable you're testing first in the condition statement.